### DISCUSSION TITLE
works on Ubuntu 24.04 LTS

### DISCUSSION THREAD

**Discussion by Poiske** [moraroy/NonSteamLaunchers-On-Steam-Deck#466] [Show and tell]
I play a lot of blizzards heartstone and after the latest update  it didn´t run on Lutris anymore. Found this solution to integrate Battle.net in steam. Works as a charm !

**Comment by moraroy** [comment]
@Poiske wow this is incredible! Thank you for letting me know I'll add it to the list! We're you able to get the plugin as well?
